vault/data_source_azure_access_credentials.go

```diff
@@ -234,7 +234,7 @@ func azureAccessCredentialsDataSourceRead(d *schema.ResourceData, meta interface
 			clientOptions.Endpoint = arm.AzurePublicCloud
 		case "AzureChinaCloud":
 			clientOptions.Endpoint = arm.AzureChina
-		case "AzureGovernmentCloud":
+		case "AzureUSGovernmentCloud":
```

Comment:
I believe this change will break anyone who has configured `AzureGovernmentCloud`, suggest adding `AzureGovernmentCloud` :

```suggestion
		case "AzureGovernmentCloud", "AzureUSGovernmentCloud":
```

Reply:
@benashz - Thank you for reviewing.

Anyone that is currently using `AzureGovernmentCloud` would receive an error from the autorest SDK:
`Error: autorest/azure: There is no cloud environment matching the name "AZUREGOVERNMENTCLOUD"`

Happy to add it if you want, but the current value is not supported and cannot be used without terraform throwing an error. (Referenced in the issue)

Reply:
Ah, yes I see, we should receive an error when getting the environment by name in this case. In that case I think we can skip this suggestion.